frontend/src/pages/list/VendorList.tsx

```diff
@@ -26,9 +27,15 @@ const COLUMNS: TableColumn[] = [
 class VendorList extends React.Component<{}, VendorListState> {
   constructor(props = {}) {
     super(props);
-    this.state = { value: "" };
+    this.state = { value: "", addVendors: "" };
   }
 
+  onSubmit = (event: FormEvent<HTMLFormElement>): void => {
+    alert("A form was submitted: \n" + this.state.addVendors);
+
+    event.preventDefault();
+  };
+
   render() {
     return <Table<VendorRow> columns={COLUMNS} data={DATA} />;
   }
```

Comment:
You should run "npm run lint" and try to fix the errors, I think most of them are due to use of "any" and the "event" not being used. I know I left a few in the codebase but we should try to have 0 (I'm going to make sure I fix the ones I introduced).